sky/tools/release_engine.py

```diff
@@ -36,42 +37,61 @@ def git_revision(cwd):
     ], cwd=cwd).strip()
 
 
-class Artifact(object):
-    def __init__(self, category, name):
-        self.category = category
-        self.name = name
+GS_ZIP_URL = 'gs://mojo/sky/shell/%(commit_hash)s/artifacts.zip'
```

Comment:
We should probably break up the zip file by platform given that we're going to add FNL and Mac artifacts that will likely be generated by separate processes (e.g., bots).